### PR TITLE
Show FluentFTP version in log after connect

### PR DESCRIPTION
**Pull request by FanDjango**
This might help in case users are secretive

```
...
Status:   Waiting for response to: SYST
Response: 215 UNIX emulated by FileZilla.
Status:   Listing parser set to: Machine
Status:   Connection established by FluentFTP Version 41.0.0.0
...
```

### CODE REVIEW

**Comment by robinrodricks** [comment]
Great, but the note seems a bit long, can we shorten it?:

- Connection established by FluentFTP Version 41.0.0.0
- Connected using FluentFTP 41.0.0.0

**Comment by FanDjango** [comment]
It is shorter now. Although they call it "verbose" for a reason, LOL

**Comment by FanDjango** [comment]
Any problem that it uses reflection?

File: FluentFTP/Client/AsyncClient/Connect.cs [hunk 1]
```diff
@@ -32,6 +33,9 @@ public partial class AsyncFtpClient {
 
 			LogFunction(nameof(ConnectAsync));
 
+			string applicationVersion = Assembly.GetAssembly(MethodBase.GetCurrentMethod().DeclaringType).GetName().Version.ToString();
+			LogWithPrefix(FtpTraceLevel.Verbose, "Connect using FluentFTP " + applicationVersion);
```
Comment (robinrodricks) [inline FluentFTP/Client/AsyncClient/Connect.cs:37]: Can you move this into BaseFtpClient by creating a reused function in the BaseFtpClient that both the clients call. This way there will be reuse of code.

File: FluentFTP/Client/SyncClient/Connect.cs [hunk 2]
```diff
@@ -33,6 +34,9 @@ public partial class FtpClient {
 
 				LogFunction(nameof(Connect));
 
+				string applicationVersion = Assembly.GetAssembly(MethodBase.GetCurrentMethod().DeclaringType).GetName().Version.ToString();
+				LogWithPrefix(FtpTraceLevel.Verbose, "Connect using FluentFTP " + applicationVersion);
+
```
Comment (robinrodricks) [inline FluentFTP/Client/SyncClient/Connect.cs:39]: same

File: FluentFTP/Client/AsyncClient/Connect.cs (hunk 1 above)
Comment (FanDjango) [reply FluentFTP/Client/AsyncClient/Connect.cs:37]: Good idea. Certainly I will do that

**Comment by robinrodricks** [comment]
Looks great!

**Pull request merged**